docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -129,13 +127,25 @@ To test your ledger in a CI pipeline, run it as part of your pipeline:
 
    .. code-block:: console
 
-     $ java -jar ledger-api-test-tool localhost:6865 2>&1 /dev/null
+     $ java -jar ledger-api-test-tool.jar localhost:6865 --all-tests --exclude=TimeIT
      $ echo $?
      0
 
 The tool is tailored to be used in CI pipelines: as customary, when the tests
 succeed, it will produce minimal output and return the success exit code.
 
+
+Output error and suite results to separate files
```

Comment:
Feel free to leave it here but I don't think we should document how to use a shell.

Reply:
removed